### PR TITLE
install-chef-suse: Add --debug switch

### PR DESCRIPTION
**Pull request by toabctl**
When running in debug mode, barclamp_install.rb is
much more verbose and also chef-client runs are providing
debugging infos.

### CODE REVIEW

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -71,6 +78,8 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
             install -p -m 0755 $tool /opt/dell/bin/
         fi
     done
+else
+    BARCLAMP_INSTALL_OPTS="$BARCLAMP_INSTALL_OPTS --rpm"
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: Hrm, did we lose the --rpm argument at some point?

**Comment by vuntz** [comment]
+1

File: scripts/install-chef-suse.sh (hunk 1 above)
Comment (toabctl) [inline scripts/install-chef-suse.sh]: Nope. ```BARCLAMP_INSTALL_OPTS``` was just overwritten if installed from git. should be fine.

Comment (vuntz) [inline scripts/install-chef-suse.sh]: Sorry, I didn't mean "lose in the patch", but we had it in the past and it seems we don't have it right now (until before your patch)?

Comment (toabctl) [inline scripts/install-chef-suse.sh]: We had it, just not for the "install from git" case

**Comment by rsalevsky** [comment]
+1

**Comment by tboerger** [comment]
:+1:

**Comment by dirkmueller** [comment]
+1

**Pull request merged**